### PR TITLE
Add more contributors

### PR DESCRIPTION
**Pull request by betatim**
Listing more people who have contributed to this project.

I went with alphabetically by surname as a way to sort people.

Yet more for #44

### CODE REVIEW

**Comment by minnsoe** [comment]
LGTM

**Comment by betatim** [comment]
Added the github handles.

**Comment by abbycabs** [comment]
Oh, I missed this! Can we highlight Chad & Zannah a bit more since they pretty much wrote the whole thing?

File: README.md [hunk 1]
```diff
@@ -28,8 +28,12 @@ This project adheres to a [code of conduct](CODE_OF_CONDUCT.md). By participatin
 
 Many thanks to the wonderful contributors who've helped create and improve this series.
```
Comment (minnsoe) [inline README.md]: Maybe something like:

Special thanks to Zannah Marsh ([@zee-moz](https://github.com/zee-moz)) and Chad Sansing ([@chadsansing](https://github.com/chadsansing)) who've played a big role in creating this series. Many thanks to the wonderful contributors who've improved this series:

**Comment by betatim** [comment]
What about now?

**Comment by abbycabs** [comment]
R+ 💯 💃 👯

**Comment by minnsoe** [comment]
🚀+1

**Comment by betatim** [comment]
What is R+? (I googled but in the day and age of personalised everything all I get is R the language)

**Comment by abbycabs** [comment]
Positive review! Sorry, some [internal lingo](https://wiki.mozilla.org/Code_Review) around code review. We finish a review with an R+/R-

**Comment by betatim** [comment]
👍 That wiki page doesn't explain r+/- I don't think, but I think I get it :)

**Pull request merged**